### ISSUE TITLE
New Website

### ISSUE THREAD

**Issue by Ombridride** [MinetestForFun/server-minetestforfun#291]
@Calinou and me have start to work on a new Website, the URL will be **secret** for the moment because the work isn't over for a "first preview" of the new Website.

#### Why ?
The actual website works on a Joomla CMS, it's horrible to manage, it's neither secure nor beautiful...

#### To Do List for the new Website
- [x] Create the Website in a dev environment
- [x] Create the 4 more important section and translate it in both FR and EN (Home, News, Download and Supprot Us)
- [x] Secure a little the website 
- [x] Create the footer with the licence
- [ ] Customize the template -> Erase the auto-title for all pages
- [ ] Vertical + horizontal center for the 4 blocks at the home ages (both FR and EN translations)
- [ ] Decide of the Menus organisation
- [ ] Add a slider, maybe "Revolution Slider" instead of the static slider of the actual template
- [ ] Do a better language switcher menu, the actual flags in the footer are very ugly
- [ ] Translate all of the French Blog articles(from the old website) in English (for the new website)
- another things SOON

**Comment by davedevils** [comment]
Website make for scratch or using other cms ?

**Comment by gravgun** [comment]
@davedevils Using Wordpress.

**Comment by Ombridride** [comment]
Yes, we use Wordpress.
For the website, we are thinking about create a web gaming platform for "libre" video games, Minetest will be a part of this website, but it's a really lot of work...
This video games platform will be interesting because i have some other game servers i can merged with this platform :)

**Comment by ObaniGemini** [comment]
Is it still up ?

**Comment by Ombridride** [comment]
Not really, this issue is more moved like a "hud gaming platform" for now.

**Comment by Ombridride** [comment]
#### Sketch design
!(sketch design)[http://i.imgur.com/OkEVrSO.png]

**Comment by ObaniGemini** [comment]
I have an other idea. What if the banners images were on the background ? like in http://terasology.org/
This would let more for other elements and immerse the user into the website. But this should be a lag creator on weak computers (as mine) I think :/

**Comment by Ombridride** [comment]
This will be really heavy, and it's a design which isn't made for text/articles reading.

We need something more adapted to "blog" type for all pages.
However, for the homepage, we can maybe do something different, maybe like you said, but for the beginning, i think we will create somehting "simple" as the sketch design show us :)

Your idea isn't a bad idea, it's just more work, and or the moment we need something over and not only a homepage :p

**Comment by Ombridride** [comment]
^ Update the initial post ^

**Comment by MightyStick** [comment]
Hi
